db/migrate/20120816192758_add_position_to_slides.rb

```diff
@@ -1,4 +1,4 @@
-class AddPositionToSlides < ActiveRecord::Migration
+class AddPositionToSlides < ActiveRecord::Migration[6.0]
   def change
     add_column :spree_slides, :position, :integer, :null => false, :default => 0
```

Comment:
could we also change ? 

```suggestion
    add_column :spree_slides, :position, :integer, null: false, default: 0
```

Reply:
fixed in https://github.com/spree-contrib/spree_slider/pull/50/files